src/VisualStudio/SetupInteractive/source.extension.vsixmanifest

```diff
@@ -4,6 +4,7 @@
     <Identity Id="A5E5EE8D-39AE-42FF-8BBF-53C5B09C07D7" Version="|%CurrentProject%;GetBuildVersion|" Language="en-US" Publisher="Microsoft" />
     <DisplayName>Roslyn Interactive Language Services</DisplayName>
     <Description>Roslyn interactive language services.</Description>
+    <ShortcutPath>..\CommonExtensions\Microsoft\ManagedLanguages\VBCSharp\InteractiveLanguageServices</ShortcutPath>
```

Comment:
I'll add an item to my backlog to merge them, but for now @jmarolf and I decided it would be easier (e.g., fewer moving parts) to first get our existing VSIX packages working.